### ISSUE TITLE
Routerseite Link zum Gateway

### ISSUE THREAD

**Issue by ChristianDresel** [FreifunkFranken/fff-monitoring#169]
auf der Routerseite (z.b. https://monitoring.freifunk-franken.de/mac/f4f26d3fbbe2 ) sind ja links unten die Gateways aufgeführt. Ein Klick auf den Link eines Gateways führt allerdings zu einer Liste der selected Router, ich glaube aber wer da drauf klickt will eher wissen was es mit dem Gateway auf sich hat. Ich fände es daher schöner wenn ein Klick auf eine Seite führt wo alle Informationen zum Gateway aufgelistet werden, wegen mir kann dann dort ein Link sein der zu den selectet Router weiterführt.

Hintergrund der Idee: 
Routerbetreiber bekommen so eher raus wer eigentlich ihr Gateway betreibt. Damit könnte etwas mehr Nähe zwischen Routeraufsteller und Gatewaybetreiber geschaffen werden, in den letzten Mails auf der ML kam raus das viele gar nicht wissen was es mit den Gateways auf sich hat oder wer die betreibt. Wir brauchen hier mehr Nähe.

**Comment by adrianschmutzler** [comment]
Ja, sehe ich inzwischen ähnlich. Die selected Liste habe ich btw noch nie benutzt.
Mal sehen, wann ich dazu komme.

Generell bin ich auch am überlegen, ob man die Gatewayinfos noch ein bisschen besser aufbereitet. Evtl. kann man pro Gateway eine eigene Seite machen, so wie bei den Usern. Mal sehen, jetzt ist mein Urlaub erstmal zu Ende, da gehen dann wieder nur noch kleine Änderungen ;-)
